<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuokattavaRaportti</class>
 <widget class="QWidget" name="MuokattavaRaportti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="pvmKehys">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sarake 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="sarake2Box">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Sarake 2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="sarake3Box">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Sarake 3</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="alkaaLabel">
        <property name="text">
         <string>Alkaa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="alkaa1Date"/>
      </item>
      <item row="1" column="2">
       <widget class="QDateEdit" name="alkaa2Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDateEdit" name="alkaa3Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="paattyyLabel">
        <property name="text">
         <string>Päättyy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="loppuu1Date"/>
      </item>
      <item row="2" column="2">
       <widget class="QDateEdit" name="loppuu2Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDateEdit" name="loppuu3Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sarake2Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>alkaa2Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake2Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>loppuu2Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake3Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>alkaa3Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake3Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>loppuu3Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake2Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>sarake3Box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake2Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>sarake3Box</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
